### DISCUSSION TITLE
Need help in overriding cmp.entry.get_documentation

### DISCUSSION THREAD

**Discussion by prashant1804** [LazyVim/LazyVim#919] [Q&A] [answered]
Is there any way to enable `cmp.entry.get_documentation` without actually installing noice.nvim plugin? I love those highlighting features.
I use the lazyvim config with noice.nvim plugin disabled.

**Comment by folke** [comment, marked as answer]
What do you mean? Those highlights come from noice. You can disable the noice cmdline, messages and popumenu. Check the noice docs

**Reply by prashant1804** [reply]
Ohh thanks, I thought the highlights were from `nvim-cmp` instead of noice.

**Reply by folke** [reply]
Noice makes lsp docs display using treesitter instead of the default ugly ones
